deployment/data-streams/changeset/v0_5_0/cs_deploy_verifier_proxy.go

```diff
@@ -0,0 +1,72 @@
+package v0_5_0
+
+import (
+	"github.com/Masterminds/semver/v3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_proxy_v0_5_0"
+)
+
+type DeployVerifierProxyConfig struct {
+	AccessControllerAddress common.Address
+}
+
+type DeployVerifierProxyImpl struct{}
+
+// DeployVerifierProxy is a changeset that deploys the verifier contract
+var DeployVerifierProxy deployment.ChangeSetV2[DeployVerifierProxyConfig] = DeployVerifierProxyImpl{}
+
+const VerifierProxy deployment.ContractType = "VerifierProxy"
+
+func (d DeployVerifierProxyImpl) Apply(e deployment.Environment, config DeployVerifierProxyConfig) (deployment.ChangesetOutput, error) {
+	newAddresses := deployment.NewMemoryAddressBook()
+
+	for chains := range e.Chains {
+		chain := e.Chains[chains]
+		_, err := deployVerifierContractEVM(
+			e.Logger, chain, newAddresses, config,
+		)
+		if err != nil {
+			return deployment.ChangesetOutput{}, err
+		}
+	}
+
+	return deployment.ChangesetOutput{AddressBook: newAddresses}, nil
+}
+
+func (d DeployVerifierProxyImpl) VerifyPreconditions(e deployment.Environment, config DeployVerifierProxyConfig) error {
+	return nil
+}
+
+func deployVerifierContractEVM(
+	lggr logger.Logger,
+	chain deployment.Chain,
+	ab deployment.AddressBook,
+	cfg DeployVerifierProxyConfig,
+) (*deployment.ContractDeploy[*verifier_proxy_v0_5_0.VerifierProxy], error) {
+	verifierProxy, err := deployment.DeployContract[*verifier_proxy_v0_5_0.VerifierProxy](lggr, chain, ab,
+		func(chain deployment.Chain) deployment.ContractDeploy[*verifier_proxy_v0_5_0.VerifierProxy] {
+			verifierProxyAddr, tx, verifierProxy, err2 := verifier_proxy_v0_5_0.DeployVerifierProxy(
+				chain.DeployerKey,
+				chain.Client,
+				cfg.AccessControllerAddress,
+			)
+
+			version := *semver.MustParse("0.5.0")
+			return deployment.ContractDeploy[*verifier_proxy_v0_5_0.VerifierProxy]{
+				Address:  verifierProxyAddr,
+				Contract: verifierProxy,
+				Tx:       tx,
+				Tv:       deployment.NewTypeAndVersion(VerifierProxy, version),
```

Comment:
I think you can reuse the version already defined [here](https://github.com/smartcontractkit/chainlink/blob/cf12b57bd2587a73432df3230ac53ed423162c40/deployment/address_book.go#L30) ?
```suggestion
				Tv:       deployment.NewTypeAndVersion(VerifierProxy, deployment.Version0_5_0),
```